modules/packages/FFTW.chpl

```diff
@@ -812,8 +812,16 @@ module FFTW {
 
     extern proc fftw_import_wisdom(read_char : fftw_read_char_func, data : c_void_ptr) : c_int;
 
+    extern proc fftw_fprint_plan(p : fftw_plan, ref output_file : c_FILE) : void;
+
+    pragma "last resort"
+    deprecated "the '_file' type is deprecated; use the variant of 'fftw_fprint_plan' that takes a 'c_FILE'"
```

Comment:
but above we have `type _file = c_FILE` ? Maybe you can just remove these and people working with `_file` should get a deprecation wherever that is created?